ui/src/auth/Purgatory.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Purgatory = () => {
+  return (
+    <div>
+      <div className="auth-page">
+        <div className="auth-box">
+          <div className="auth-logo" />
+          <div className="auth--purgatory">
+            <h3>
+              Logged in to <strong>[ORGNAME]</strong>
+            </h3>
+            <p>
+              You have not been assigned a Role yet<br />Contact your
```

Comment:
it's not that you have not been assigned a role, it's that your role lacks sufficient access to see anything.